ocs_ci/ocs/ui/views.py

```diff
@@ -2133,6 +2133,34 @@
         "//*[@id='yaml-create']",
         By.XPATH,
     ),
+    "pagination_next_button": (
+        "button[class='pf-v5-c-button pf-m-plain pf-v5-u-ml-xs'] svg",
+        By.CSS_SELECTOR,
+    ),
+    "pagination_prev_button": (
+        "button[class='pf-v5-c-button pf-m-plain pf-v5-u-mr-xs'] svg",
+        By.CSS_SELECTOR,
+    ),
+    "bucket_list_items": (
+        "//a[starts-with(@href, '/odf/object-storage/buckets/')]",
+        By.XPATH,
+    ),
+    "bucket_action_button": (
+        "tbody tr:nth-child(1) td:last-child button:first-child",
+        By.CSS_SELECTOR,
+    ),
+    "bucket_delete_option": (
+        "tbody li:nth-child(2) button:nth-child(1) span:nth-child(1) span:nth-child(1)",
```

Comment:
these locators are very fragile, any change in order of dropdown items will brake the test

Reply:
what would you suggest ?

Reply:
usually we find locators that are unique, or build them through axe of the closest element that is unique
https://www.w3schools.com/xml/xpath_axes.asp